addons/im_livechat/models/mail_channel.py

```diff
@@ -98,6 +99,30 @@ def channel_fetch_slot(self):
         values['channel_livechat'] = self.search([('channel_type', '=', 'livechat'), ('id', 'in', pinned_channels.ids)]).channel_info()
         return values
 
+    def _channel_get_livechat_website_visitor_info(self):
+        if self.livechat_operator_id in self.channel_partner_ids:
```

Comment:
I'm not sure this condition is needed, if the operator somehow left the channel and there is still someone else inside, there is no need to fallback on anonymous name.

Reply:
Just mimicking `_channel_get_livechat_partner_name`, so I just pick 1st partner in `channel_partner_ids` if there not `livechat_operator_id`, right?

Reply:
No I mean you can just get rid of the condition here.
The rest of the code is fine, you pick the first `channel_partner_ids` that is not the `livechat_operator_id` (due to the `-`), which is what is done.